frontend/src/scenes/surveys/Surveys.tsx

```diff
@@ -146,7 +141,8 @@ export function Surveys(): JSX.Element {
                         }}
                         className="mb-2"
                     >
-                        Survey popovers are currently disabled for this project.
+                        Survey popovers are currently disabled for this project but there are active surveys running.
+                        Re-enable them in the settings.
```

Comment:
link to settings?

Reply:
It's linked in the configure button right in the banner 😃